test/e2e/autoscaling/cluster_autoscaler_scalability.go

```diff
@@ -313,6 +313,28 @@ var _ = framework.KubeDescribe("Cluster size autoscaler scalability [Slow]", fun
 		Expect(len(nodes.Items)).Should(Equal(totalNodes))
 	})
 
+	It("should do nothing when pending pods are unschedulable [Feature:ClusterAutoscalerScalability6]", func() {
+		// Start a number of pods saturating existing nodes.
+		initialPodReplicas := nodeCount * 2
+		perNodeReservation := int(float64(memCapacityMb) * 0.20)
+		initialPodsTotalMemory := nodeCount * perNodeReservation
+		reservationCleanup := ReserveMemory(f, "initial-pod", initialPodReplicas, initialPodsTotalMemory, true /* wait for pods to run */, scaleUpTimeout)
```

Comment:
Fair enough, it makes sense to use shorter timeout for that. Though I would prefer to have a separate const with less confusing name for it.